DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
```

Comment:
You should edit the text above, because we now have a solution!

I think it's still good to explain **why** it's not trivial to interact with an IDE, due to the file paths mismatch.